disposable_email_blacklist.conf

```diff
@@ -75,6 +80,9 @@
 4warding.com
 4warding.net
 4warding.org
+5.emailfake.ml
```

Comment:
Not needed anymore as `emailfake.ml` will replace the need to declare `5.emailfake.ml`